### PR TITLE
Add ABE for <amp-facebook-page>

### PR DESCRIPTION
**Pull request by nainar**

### CODE REVIEW

File: src/20_Components/amp-facebook-page.html [hunk 1]
```diff
@@ -0,0 +1,75 @@
+<!--
+  ## Introduction
+
+  The amp-facebook-page component allows embedding Facebook pages into AMP files.
```
Comment (sebastianbenz) [inline src/20_Components/amp-facebook-page.html]: Can you please link to the official docs.

File: src/20_Components/amp-facebook-page.html [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+    <script async custom-element="amp-facebook-page" src="https://cdn.ampproject.org/v0/amp-facebook-page-0.1.js"></script>
+  <link rel="canonical" href="<%host%>/components/amp-facebook-page/">
+  <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+  <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+  </head>
+  <body>
+
+    <!-- ## Basic Usage -->
+    <!-- The `amp-facebook-page` component only requires the href of the facebook post... -->
```
Comment (sebastianbenz) [inline src/20_Components/amp-facebook-page.html]: nit: s/..././

File: src/20_Components/amp-facebook-page.html [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+    <amp-facebook-page width="340" height="130"
+        layout="responsive"
+        data-href="https://www.facebook.com/imdb/">
+    </amp-facebook-page>
+
+    <!-- ## Customizing the embed -->
+    <!-- The `amp-facebook-page` component allows you to customize the embed via several attributes: -->
+
+    <!-- **Example 1: You can set the locale to render the page in a different language, for example French.** -->
```
Comment (sebastianbenz) [inline src/20_Components/amp-facebook-page.html]: Suggestion: you always prefix subsample with "Example X". I like this approach, however, we don't do this for the other samples. Could you, for consistency reasons, use subheadings instead. For example:

```
 <!-- 
### Different Languages

You can set the locale to render the page in a different language, for example French. 
-->
...
```

This way the different samples show up in the menu to the left.

File: src/20_Components/amp-facebook-page.html (hunk 1 above)
Comment (nainar) [reply src/20_Components/amp-facebook-page.html]: Done

File: src/20_Components/amp-facebook-page.html (hunk 2 above)
Comment (nainar) [reply src/20_Components/amp-facebook-page.html]: Done.

File: src/20_Components/amp-facebook-page.html (hunk 3 above)
Comment (nainar) [reply src/20_Components/amp-facebook-page.html]: Done.

File: src/20_Components/amp-facebook-page.html [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+    <!-- ## Show different tabs -->
+    <!-- **Render the `timeline` and `events` tab for a particular Facebook page.** -->
+    <amp-facebook-page width="552" height="310"
+        layout="responsive"
+        data-tabs="timeline, events"
+        data-href="https://www.facebook.com/barackobama/">
+    </amp-facebook-page>
+
+    <!-- ## Facebook page cover -->
+    <!-- **Example 3: Hide the cover of a particular Facebook page.** -->
```
[the remark this answers is not in the document]
Comment (nainar) [reply src/20_Components/amp-facebook-page.html]: Done.

**Comment by nainar** [comment]
@sebastianbenz please take a look?

File: src/20_Components/amp-facebook-page.html [hunk 5]
```diff
@@ -30,47 +30,47 @@
...
+    <!-- Render the `timeline` and `events` tab for a particular Facebook page by specifying  `data-tabs="timeline, events"`. -->
     <amp-facebook-page width="552" height="310"
         layout="responsive"
         data-tabs="timeline, events"
         data-href="https://www.facebook.com/barackobama/">
     </amp-facebook-page>
 
     <!-- ## Facebook page cover -->
-    <!-- **Example 3: Hide the cover of a particular Facebook page.** -->
+    <!-- Example 3: Hide the cover of a particular Facebook page by specifying `data-hide-cover="true"`. -->
```
[the remark this answers is not in the document]
Comment (nainar) [reply src/20_Components/amp-facebook-page.html]: Sorry for leaving those out - Done!

**Pull request merged**